app/views/clients/_show_navigation.html.slim

```diff
@@ -1,10 +1,10 @@
-
 nav.navbar.section-navigation
   ul.list-inline
-    li= button_link t_action(:edit), edit_client_path(@client)
-    - if policy(Journal).index?
-      li= button_link 'Journal', polymorphic_path([@client, Journal])
+    li= button_link t_action(:edit), edit_client_path(@client), 'success'
+    li= button_link t_action(:print), client_path(@client, print: true)
     - if policy(Client).destroy?
       li= link_to t_action(:destroy), @client, confirm_deleting(@client, 'btn btn-default')
     li.m-l-20= button_link t('back'), clients_path
```

Comment:
I think this `m-l-20` is not needed anymore(there is a space before the back button)
<img width="342" alt="screen shot 2017-11-03 at 18 25 05" src="https://user-images.githubusercontent.com/11731325/32387306-577b8e62-c0c4-11e7-9c40-c8b711e16a91.png">

Reply:
Changed